tests/events/18-suite-event-mail.t

```diff
@@ -16,6 +16,7 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #-------------------------------------------------------------------------------
 # Test event mail.
+CYLC_TEST_IS_GENERIC=false
```

Comment:
It should just need the `mail` command, but if you can't get it to work under Travis CI, that's another story.

The mock SMTP server is set up by line 25-36, which calls `mock_smtpd_init` in our `test_header` script and sets up a `global.rc` with the right values. Perhaps this logic does not work correctly under Travis CI?